### PR TITLE
Bugs/AdvancedServer/CLK32-Fixbug Advanced Server

### PR DESCRIPTION
**Pull request by Minhdn1**
## Prerequisite
- [x] Self-review :monocle_face:
- [x] No Crashes ☠️
---

### Root cause
- // Explain what you have done

### Solution
- // Explain how you fix it

### Impact
- // List of affected components/modules

### Screenshots (Optional)
- // Capture affected screens

### CODE REVIEW

File: ClearKeep/App/Resources/Localize.strings [hunk 1]
```diff
@@ -20,6 +20,7 @@
 "General.Email.Valid" = "Email is invalid";
 "General.Password.Valid" = "Password must be between 6-12 characters";
 "General.ConfirmPassword.Valid" = "Password and Confirm password do not match. Please try again";
+"General.Closse" = "Closse";
```
Comment (vmo-klinh) [inline ClearKeep/App/Resources/Localize.strings]: General.'Closse': typo

File: ClearKeep/App/Resources/Localize.strings [hunk 2]
```diff
@@ -129,6 +130,8 @@
 "AdvancedServer.Submit" = "Submit";
 "AdvancedServer.ServerUrl" = "Server URL";
 "AdvancedServer.Alert" = "You are using custom server";
+"AdvancedServer.Title.Error" = "Display Error";
+"AdvancedServer.Message.Error" = "Wrong server URL.Please try again";
```
Comment (vmo-klinh) [inline ClearKeep/App/Resources/Localize.strings]: Missing space

**Review by vmo-klinh — CHANGES_REQUESTED** [review]

File: ClearKeep/App/Resources/Localize.strings (hunk 1 above)
Comment (Minhdn1) [reply ClearKeep/App/Resources/Localize.strings]: fix complete

File: ClearKeep/App/Resources/Localize.strings (hunk 2 above)
Comment (Minhdn1) [reply ClearKeep/App/Resources/Localize.strings]: fix complete

**Review by vmo-klinh — APPROVED** [review]

**Pull request merged**